### PR TITLE
Various fixes/additions

### PR DESCRIPTION
**Pull request by Eejit43**
Yay, GitHub finally works again!

### CODE REVIEW

File: PET_SKIN_WOLF.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{
+  "itemid": "minecraft:skull",
+  "displayname": "§5Dark Wolf Skin",
+  "nbttag": "{overrideMeta:1b,HideFlags:254,SkullOwner:{Id:\"be4a9ed9-609c-32ea-b49b-44a7bd7ab0f2\",Properties:{textures:[0:{Value:\"eyJ0ZXh0dXJlcyI6eyJTS0lOIjp7InVybCI6Imh0dHA6Ly90ZXh0dXJlcy5taW5lY3JhZnQubmV0L3RleHR1cmUvYzhlNDE0ZTc2MmUxMDI0Yzc5OWU3MGI3YTUyN2MyMmZiOTU2NDhmMTQxZDY2MGIxMGM1MTJjYzEyNDMzNDIxOCJ9fX0K\"}]}},display:{Lore:[0:\"§8Consumed on use\",1:\"\",2:\"§7§7Pet skins change the look and\",3:\"§7particle trail of your pet but\",4:\"§7only one skin can be active at a\",5:\"§7time!\",6:\"\",7:\"§7§7This skin can only be applied\",8:\"§7to Wolf pets.\",9:\"\",10:\"§7§eRight click on your pet to\",11:\"§eapply this skin!\",12:\"\",13:\"§5§lEPIC COSMETIC\"],Name:\"§5Dark Wolf Skin\"},ExtraAttributes:{originTag:\"FIRE_SALE\",id:\"PET_SKIN_WOLF\",uuid:\"5e6f7669-2260-4cbc-a656-57a37fe8b6e2\",timestamp:\"8/10/21 11:28 AM\"},AttributeModifiers:[]}",
```
Comment (jani270) [inline PET_SKIN_WOLF.json:4]: You didnt remove the UUID and Timestamp here

File: items/WISHING_COMPASS.json [hunk 2]
```diff
@@ -1,26 +1,17 @@
...
+  "nbttag": "{HideFlags:254,SkullOwner:{Id:\"b79aa74e-f5ac-3d6b-a9a8-169b227f9699\",Properties:{textures:[0:{Value:\"eyJ0ZXh0dXJlcyI6eyJTS0lOIjp7InVybCI6Imh0dHA6Ly90ZXh0dXJlcy5taW5lY3JhZnQubmV0L3RleHR1cmUvYjhhMDY1NTg4YzdkYmYxMjk0NTk1YWJhNzc3OTFjM2FkNjVmMTliZjFjOWFkN2E1YzIzZGE0MGI4Mjg2MGI3In19fQ\u003d\u003d\"}]}},display:{Lore:[0:\"§7§7§oTrue enough, this compass\",1:\"§7§odoes not point north. It points\",2:\"§7§oto the thing you want most in\",3:\"§7§othis world.\",4:\"\",5:\"§7§7The §acompass §7will point to\",6:\"§7§dimportant locations §7when in\",7:\"§7the §bCrystal Hollows§7.\",8:\"\",9:\"§7Remaining Uses: §e3§8/§e3\",10:\"\",11:\"§a§lUNCOMMON\"],Name:\"§aWishing Compass\"},ExtraAttributes:{originTag:\"UNKNOWN\",id:\"WISHING_COMPASS\",uuid:\"c8c46ef7-ae53-45c2-a75e-39933d2aae7d\",timestamp:\"7/14/21 10:07 PM\"}}",
```
Comment (jani270) [inline items/WISHING_COMPASS.json:4]: You didnt remove the UUID and Timestamp here

File: items/REFINED_DIAMOND.json [hunk 3]
```diff
@@ -1,26 +1,20 @@
...
   "displayname": "§5Refined Diamond",
-  "nbttag": "{ench:[],overrideMeta:1b,HideFlags:254,display:{Lore:[0:\"\",1:\"§7§7§oLooking at it for too long\",2:\"§7§omay cause eye trauma.\",3:\"\",4:\"§5§lEPIC\",5:\"\",6:\"§eItems Required\",7:\"§9Enchanted Diamond Block §7x2\",8:\"\",9:\"§7Duration: §b8h\"],Name:\"§5Refined Diamond\"},ExtraAttributes:{id:\"REFINED_DIAMOND\"},AttributeModifiers:[]}",
+  "nbttag": "{ench:[],HideFlags:254,display:{Lore:[0:\"§7§7§oLooking at it for too long\",1:\"§7§omay cause eye trauma.\",2:\"\",3:\"§5§lEPIC\"],Name:\"§5Refined Diamond\"},ExtraAttributes:{id:\"REFINED_DIAMOND\"}}",
   "damage": 0,
   "lore": [
-    "",
     "§7§7§oLooking at it for too long",
     "§7§omay cause eye trauma.",
     "",
-    "§5§lEPIC",
```
Comment (jani270) [inline items/REFINED_DIAMOND.json:11]: Why did you remove the recipe, its still a forge item

**Review by jani270 — CHANGES_REQUESTED** [review]

**Comment by Eejit43** [comment]
Should be fixed, sorry!

File: items/REFINED_DIAMOND.json [hunk 4]
```diff
@@ -13,14 +12,16 @@
     "§eItems Required",
     "§9Enchanted Diamond Block §7x2",
     "",
-    "§7Duration: §b8h"
+    "§7Duration: §b8h",
+    "",
```
Comment (jani270) [inline items/REFINED_DIAMOND.json:16]: Line 16 and 17 are not needed

**Review by jani270 — CHANGES_REQUESTED** [review]

**Comment by Eejit43** [comment]
Alright, all should be good now, sorry that took so many attempts...

**Review by jani270 — APPROVED** [review]
Seems fine

**Review by IRONM00N — APPROVED** [review]
I think it looks good, too lazy to test tho so I hope nothing breaks

**Pull request merged**